@danielballan, I don't understand why Travis changed Python's directories in [this build](https://travis-ci.org/NSLS-II/docs/builds/444855172?utm_source=github_status&utm_medium=notification):
```
/home/travis/build/NSLS-II/docs/source/_cookbook/live_recon.py failed to execute correctly: Traceback (most recent call last):
  File "/home/travis/build/NSLS-II/docs/source/_cookbook/live_recon.py", line 36, in <module>
    import tomopy
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/tomopy/__init__.py", line 55, in <module>
    from tomopy.misc.corr import *
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/tomopy/misc/corr.py", line 60, in <module>
    import tomopy.util.extern as extern
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/tomopy/util/extern.py", line 115, in <module>
    LIB_TOMOPY = c_shared_lib('libtomopy')
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/tomopy/util/extern.py", line 109, in c_shared_lib
    return load_dll(sharedlib)
  File "/opt/python/3.6.3/lib/python3.6/ctypes/__init__.py", line 426, in LoadLibrary
    return self._dlltype(name)
  File "/opt/python/3.6.3/lib/python3.6/ctypes/__init__.py", line 348, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: libmkl_rt.so: cannot open shared object file: No such file or directory
```

In the first case it's `/home/travis/virtualenv/python3.6.3/lib/python3.6`, but then `/opt/python/3.6.3/lib/python3.6/`. Any ideas?

I have a workaround in https://github.com/NSLS-II/docs/pull/61/commits/41141c8b92bfe525e83fad2a0059ca6017b9d3b4#diff-354f30a63fb0907d4ad57269548329e3R17, which seems to solve the problem, but not sure why it's not searching in the proper place first (where Python venv is installed).

--- a/source/_cookbook/live_recon.py
+++ b/source/_cookbook/live_recon.py
@@ -8,7 +8,7 @@ Problem
 Reconstruct a 2D cross-section of a sample from projections, as if it were
 rotated in place in front of a strip detector.
 
-This example uses the library `tomopy <tomopy.readthedocs.io>`_.
+This example uses the library `tomopy <https://tomopy.readthedocs.io>`_.
 
 Approach
 --------
@@ -22,8 +22,7 @@ Now, use tomopy to reconstruct the sample "density" from these projections.
 To give tomopy live access to the data, wrap it in a class that acts as a
 bluesky callback, an interface between tomopy and bluesky's document model.
 
-Also make a callback for visualizing the projections side by side (a
-sinogram).
+Also make a callback for visualizing the projections side by side (a sinogram).
 
 Example Solution
 ----------------

